README.md

```diff
@@ -110,89 +152,65 @@ execute the following in any OCaml top-level:
 ```ocaml
 # #use "topfind";;
 # #require "bap.top";;
-# open Bap.Std;;
 ```
 
-And everything should work just out of box, i.e. it will load all the
-dependencies, install top-level printers, etc.
+## Shell
 
+The BAP main frontend is a command line utility called
+[bap](https://github.com/BinaryAnalysisPlatform/bap-python). You can
```

Comment:
Is this correct? I don't think that `bap-python` is where the bap frontend utility comes from.